sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestUserMessage.java

```diff
@@ -148,7 +145,17 @@ public static ChatRequestUserMessage fromJson(JsonReader jsonReader) throws IOEx
                 String fieldName = reader.getFieldName();
                 reader.nextToken();
                 if ("content".equals(fieldName)) {
-                    content = reader.getNullable(nonNullReader -> BinaryData.fromObject(nonNullReader.readUntyped()));
+                    if (reader.currentToken() == JsonToken.STRING) {
+                        content = BinaryData.fromString(reader.getString());
+                    } else if (reader.currentToken() == JsonToken.START_OBJECT) {
+                        content = BinaryData.fromObject(reader.readMap(JsonReader::readUntyped));
+                    } else if (reader.currentToken() == JsonToken.START_ARRAY) {
+                        content = BinaryData.fromObject(reader.readArray(JsonReader::readUntyped));
+                    } else if (reader.currentToken() == JsonToken.NULL) {
+                        content = null;
```

Comment:
Are string, object and arrays the only types of content we can have?

Reply:
Yeah. We have String, arrays(Array and List), Object(BinaryDatra) only 

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestUserMessage.java#L35-L70